frontend/src/pages/projects/screens/detail/DetailsSection.tsx

```diff
@@ -43,7 +43,7 @@ const DetailsSection: React.FC<DetailsSectionProps> = ({
 
     if (isLoading) {
       return (
-        <Bullseye>
+        <Bullseye style={{ minHeight: 150 }}>
```

Comment:
Hack so the spinner gets a bit of space -- not sure if this is the best way to go about it -- but I find if we don't, Pipelines juts all kinds around during setup. Imo it looks better for the other sections loading too.